Add example case for sending an expression that contains a variable

ystaggl opened the pull request:
Added an example for sending an expression that contains a variable

ystaggl committed:
Added example case for sending an expression.

Right now it's completely unclear how you can use expressions with Send. I think it's a very powerful piece of functionality so it deserves to have it's own example at minimum.

I added an example case, ExExtended, to demonstrate this functionality.

ystaggl committed:
Update Send.htm

removed redundant line

ystaggl committed:
modify example case

modified the example case to focus more on the variable part of it.

ystaggl committed:
fixed href

forgot to change href from ExExpression to ExVarExpression in last commit. That's now fixed.

ystaggl committed:
reworded description

Ragnar-F commented:
"Sends an expression" will cause confusion because everything in v2 is expression syntax (there are a few exceptions such as directive parameters). If you remove everything regarding expression, I'm willing to merge this. An example for variable usage makes sense, especially for beginners.

ystaggl committed:
Removed references to expressions.

Renamed the example and changed the description to remove any references to expression, per Ragnar-F's advice.

ystaggl commented:
Done! Thanks for your input.
